.travis.yml

```diff
@@ -35,13 +55,23 @@ jobs:
       # Docs and lint python versions must be synced with those in tox.ini
     - python: '3.8'
       env: TOX_ENV=docs
+    - python: '3.8'
+      arch: arm64
+      env: TOX_ENV=docs
     - python: '3.8'
       env: TOX_ENV=lint
   allow_failures:
     # Currently failing due to https://foss.heptapod.net/pypy/cffi/issues/458
     - python: nightly
 
 install:
+    - |
+      if [[ -n ${BUILD_WHEEL} ]]; then
+         pip install -U cibuildwheel
+         cibuildwheel --output-dir dist
+         rm -rf dist/*manylinux2010_x86_64.whl
+         pip install dist/*.whl
```

Comment:
`tox` runs all the tests in an isolated virtualenv, so installing the wheel here doesn't cause it to get tested. I think you need to tell tox to use the wheel that was just built.